P0009/P0009.bs

```diff
@@ -1796,12 +1796,12 @@ Table �: Accessor policy requirements
 <tr>
   <td>`A::offset_policy`</td>
   <td></td>
-  <td>Accessor policy compatible with returning a offseted pointer.</td>
+  <td>Accessor policy which can return a pointer that has been offset.</td>
```

Comment:
Shouldn't this be `which can access` rather than `which can return`?  Also, we need the restriction that it can be constructed from a const reference to an instance of `A`

Reply:
"Meets the requirements of `${CONCEPT}` concept" ?

Reply:
"which can access a pointer which is returned by `a.offset`"